install/host_preparation.adoc

```diff
@@ -168,6 +172,13 @@ Alternatively, disable all repositories:
  yum-config-manager --disable \*
 ----
 +
+[NOTE]
```

Comment:
I think it would be more helpful to add a separate step to install the yum-config-manager package before this step.